docs/admin/disruptions.md

```diff
@@ -0,0 +1,87 @@
+---
+assignees:
+- mml
+
+---
+This guide is for anyone wishing to specify safety constraints on pods or anyone
+wishing to write software (typically automation software) that respects those
+constraints.
+
+* TOC
+{:toc}
+
+## Rationale
+
+Various cluster management operations may voluntarily evict pods.  "Voluntary"
+means an eviction can be safely delayed for a reasonable period of time. The
+principal examples today are draining a node for maintenance or upgrade
+(`kubectl drain`), and cluster autoscaling down. In the future the
+[rescheduler](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/rescheduling.md)
+may also perform voluntary evictions.  By contrast, something like evicting pods
+because a node has become unreachable or reports NotReady, is not "voluntary."
+
+For voluntary evictions, it can be useful for applications to be able to limit
+the number of pods that are down, or the rate with which pods are evicted.  For
```

Comment:
Since we didn't implement it yet, I would just remove the "or the rate with which pods are evicted"